### ISSUE TITLE
Phpunit tests fails on 64-bit systems

### ISSUE THREAD

**Issue by Powerhead13** [fzaninotto/Faker#982]
I'm seeing 2 tests fails on Win 10 64 bit.
This definitely happens because of PHP integer overload on 64 bit systems.
This can be fixed by adding quotes to dataprovider values in `LuhnTest` to convert em to a strings.
Especially, values `561059108101825 `and `601100099013942`
```
There were 2 failures:

1) Faker\Test\Calculator\LuhnTest::testComputeCheckDigit with data set #9 (561059108101825, '0')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'0'
+'7'

E:\Projects\faker-fork\test\Faker\Calculator\LuhnTest.php:34

2) Faker\Test\Calculator\LuhnTest::testComputeCheckDigit with data set #10 (601100099013942, '4')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'4'
+'3'
```

**Comment by Powerhead13** [comment]
I created PR to fix that https://github.com/fzaninotto/Faker/pull/983

**Comment by fzaninotto** [comment]
Fixed by #983
